src/EnergyPlus/EconomicLifeCycleCost.cc

```diff
@@ -2649,22 +2418,22 @@ void WriteTabularLifeCycleCostReport(EnergyPlusData &state)
         rowHead(16) = "Grand Total";
         columnHead(1) = "Present Value";
 
-        tableBody(1, 1) = RealToStr(elcc->CashFlow(costCatConstruction).presentValue, 2);
-        tableBody(1, 2) = RealToStr(elcc->CashFlow(costCatSalvage).presentValue, 2);
-        tableBody(1, 3) = RealToStr(elcc->CashFlow(costCatOtherCapital).presentValue, 2);
-        tableBody(1, 4) = RealToStr(elcc->CashFlow(costCatEnergy).presentValue, 2);
-        tableBody(1, 5) = RealToStr(elcc->CashFlow(costCatWater).presentValue, 2);
-        tableBody(1, 6) = RealToStr(elcc->CashFlow(costCatMaintenance).presentValue, 2);
-        tableBody(1, 7) = RealToStr(elcc->CashFlow(costCatRepair).presentValue, 2);
-        tableBody(1, 8) = RealToStr(elcc->CashFlow(costCatOperation).presentValue, 2);
-        tableBody(1, 9) = RealToStr(elcc->CashFlow(costCatReplacement).presentValue, 2);
-        tableBody(1, 10) = RealToStr(elcc->CashFlow(costCatMinorOverhaul).presentValue, 2);
-        tableBody(1, 11) = RealToStr(elcc->CashFlow(costCatMajorOverhaul).presentValue, 2);
-        tableBody(1, 12) = RealToStr(elcc->CashFlow(costCatOtherOperational).presentValue, 2);
-        tableBody(1, 13) = RealToStr(elcc->CashFlow(costCatTotEnergy).presentValue, 2);
-        tableBody(1, 14) = RealToStr(elcc->CashFlow(costCatTotOper).presentValue, 2);
-        tableBody(1, 15) = RealToStr(elcc->CashFlow(costCatTotCaptl).presentValue, 2);
-        tableBody(1, 16) = RealToStr(elcc->CashFlow(costCatTotGrand).presentValue, 2);
+        tableBody(1, 1) = RealToStr(elcc->CashFlow[CostCategory::Construction].presentValue, 2);
```

Comment:
Can be done with a loop rather than unrolled code.

Reply:
Again, the indexing order is a bit different from the enum, so making this into a loop will cause table diffs. I will leave this to the end to show that this is what is causing the diffs. Or do it in a different PR.